### PR TITLE
added solr instructions and fixed typo in branch naming convention

### PR DESCRIPTION
**Pull request by kkrebs**

### CODE REVIEW

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 07655404a1694415e8827b40fc83192114142fab! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/2023-pull_request-367

File: README.md [hunk 1]
```diff
@@ -4,13 +4,18 @@ MIR (acronym for MyCoRe/MODS Institutional Repository) is an open source reposit
 
 ## Installation instructions
 Detailed instructions you can find on [MIR Documentation site](http://mycore.de/generated/mir)
- - starting up a servlet container in development environment is quite simple: `mvn install -am -pl mir-webapp && mvn -Pdev -Djetty org.codehaus.cargo:cargo-maven2-plugin:run -pl mir-webapp` If you want to test the application with Tomcat instead replace `-Djetty` by `-Dtomcat=9`
  - initialize solr configuration using `git submodule update --init --recursive`
+ - to start solr, go to mir-webapp
+  - install solr with the command: `mvn solr-runner:copyHome solr-runner:installSolrPlugins`
```
Comment (paschty) [inline README.md]: solr-runner:installSolrPlugins ist not necessary anymore.

File: README.md [hunk 2]
```diff
@@ -4,13 +4,18 @@ MIR (acronym for MyCoRe/MODS Institutional Repository) is an open source reposit
...
+  - run solr with the command: `mvn solr-runner:start`
+  - stop solr with the command: `mvn solr-runner:stop`
+  - update solr with the command: `mvn solr-runner:stop solr-runner:copyHome solr-runner:start`
+ - starting up a servlet container in development environment is quite simple: `mvn install -am -pl mir-webapp && mvn -Pdev -Djetty org.codehaus.cargo:cargo-maven2-plugin:run -pl mir-webapp` If you want to test the application with Tomcat instead replace `-Djetty` by `-Dtomcat=9`
```
Comment (paschty) [inline README.md]: Missing info: you need to run this command from mir folder(instead of mir-webapp)

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 04279cfaa03b97d1b4d6aa2bbdbc5eae5ae71e89! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/2028-pull_request-367

**Pull request merged**